### PR TITLE
Added the ability to print a virus release form using the PANDEMIC

### PR DESCRIPTION
**Pull request by farie82**
Adds the ability to print a virus release form using the PANDEMIC

1. Will check if the virus is advanced and named. If so add the option
2. When the button is pressed it'll ask for a reason to release. (Paper will have a field to add more to the reason if a mistake was made)
3. Then it'll ask if the user wants to sign the document himself.

Example (does not print prestamped, if only)
image

Props to Urbanliner for his virus form format.

Since it's a feature freeze, but a smol feature HUE. now I'll use this fix:
#9633

:cl: Farie82
add: PANDEMIC can print a virus release form now. Viromancers REJOICE!
/:cl:

### CODE REVIEW

**Comment by farie82** [comment]
Current format is directly copied from the wiki page. I'll change it though since you're right. It'll look better with the line below it and the sign part on a separate line.

**Comment by farie82** [comment]
Edit. If you stamp it'll create a line on it's own. So I'll leave the line out. The new line is nice to have though.
Now(next version has it without the line)
image

File: code/modules/reagents/chemistry/machinery/pandemic.dm [hunk 1]
```diff
@@ -172,6 +177,46 @@
...
 	return
 
+//Prints a nice virus release form. Props to Urbanliner for the layout
+/obj/machinery/computer/pandemic/proc/print_form(var/datum/disease/advance/D, mob/living/user)
+	D = archive_diseases[D.GetDiseaseID()]
+	if(!(printing) && D)
+		var/reason = input(user,"Enter a reason for the release", "Write", null) as message
+		reason += "<span class=\"paper_field\"></span>"
+		var/english_symptoms = list()
+		for(var/datum/symptom/S in D.symptoms)
```
Comment (Birdtalon) [inline code/modules/reagents/chemistry/machinery/pandemic.dm]: Assuming you know that D.symptoms only contains objects of type `/datum/symptom` you can use a typeless loop here which is faster.

```
for(var/I in D.symptoms)
    var/datum/symptom/D = I
   ...etc()
```

Comment (farie82) [reply code/modules/reagents/chemistry/machinery/pandemic.dm]: Nice!
Didn't know it would be faster that way

Comment (Birdtalon) [reply code/modules/reagents/chemistry/machinery/pandemic.dm]: It's faster because you avoid an `istype()` check altogether.

**Comment by variableundefined** [comment]
PR approved

**Pull request merged**